Guides/Modeling and Binding.md

```diff
@@ -0,0 +1,584 @@
+# Modeling and Binding
+
+This guide will walk you through a practical example of taking an app spec/design and turning it into a working `IGListKit` project.
+
+You will learn how to:
+
+- Turn a design spec into a top-level model and view models
+- Use `IGListBindingSectionController` for animated, one-way cell updates
+- Cell-to-controller action handling and delegation
+- Updating the UI with local data mutations
+
+## Getting Started
+
+You can follow along and build the example in this guide. First, you must download [this starter project](https://github.com/rnystrom/IGListKit-Binding-Guide). Open **ModelingAndBinding-Starter/ModelingAndBinding.xcworkspace** since the base project is setup with CocoaPods with `IGListKit` already added as a dependency.
+
+Take a look at the following Instagram-inspired list element design:
+
+![Design Specs](../Resources/modeling-design.png)
+
+You can already start mentally modelling your data:
+
+- The top cell has a **username** and **timestamp** label
+- The image cell will need some sort of image `URL`
+- An action cell with **like count**. There will also need to be some sort of action handling when someone taps the heart
+- Then there are a _dynamic_ number of comment cells that contain a **username** and **comment**
+
+Remember that `IGListKit` functions on **one model per section controller**. All of the cells in this design correlate to one top-level "post" object delivered by a server. You want to create a `Post` model that contains all of the information that the cells require.
+
+> A common mistake is to create a single model and section controller for a single cell. In this example, that will create a **very confusing** architecture since the top-level objects will contain a mix and match of user, image, and action, and comment models.
```

Comment:
`will contain a mix and match of user, image, action, and comment models.` (remove one "and")